### PR TITLE
added cli command for generating condensed schemapacks (GSI-697)

### PR DESCRIPTION
**Pull request by KerstenBreuer**
```
- write all logging info to stderr so that stdout
  is reserved for primary output
- use "rt" typ instead of "safe" for ruamel yaml
  to preserve output order (e.g. "schemapack:"
  should show up at the top)
- add a function to dump a schemapack as a condensed string

Bumps version to 2.0.0-alpha.4
```

### CODE REVIEW

File: src/schemapack/_internals/dump.py [hunk 1]
```diff
@@ -17,14 +17,15 @@
...
+from io import StringIO
 from pathlib import Path
 from typing import Any
 
 import ruamel.yaml
 
 from schemapack._internals.spec.schemapack import SchemaPack
 
-yaml = ruamel.yaml.YAML(typ="safe")
+yaml = ruamel.yaml.YAML(typ="rt")
```
Comment (lkuchenb) [inline src/schemapack/_internals/dump.py:28]: Do we still need to set `default_flow_style=False` then?

File: tests/test_cli.py [hunk 2]
```diff
@@ -36,7 +39,10 @@
     VALID_SCHEMAPACK_PATHS,
 )
 
-runner = CliRunner(mix_stderr=False)
+yaml = ruamel.yaml.YAML(typ="rt")
```
Comment (lkuchenb) [inline tests/test_cli.py:42]: Do you use `yaml` anywhere?

**Review by lkuchenb — APPROVED** [review]
Thanks, only two minor comments

File: src/schemapack/_internals/dump.py (hunk 1 above)
Comment (KerstenBreuer) [reply src/schemapack/_internals/dump.py:28]: Tests also run without. I removed it.

**Review by lkuchenb — APPROVED** [review]

**Pull request merged**